data_structures/stacks/next_greater_element.py

```diff
@@ -1,24 +1,54 @@
-def printNGE(arr):
+def next_greatest_element_slow(arr):
     """
-    Function to print element and Next Greatest Element (NGE) pair for all elements of list
-    NGE - Maximum element present afterwards the current one which is also greater than current one
+    Function to get Next Greatest Element (NGE) pair for all elements of list
+    Maximum element present afterwards the current one which is also greater than current one
     >>> printNGE([11,13,21,3])
```

Comment:
```suggestion
    >>> next_greatest_element_slow([11,13,21,3])
```

Reply:
Thanks for your response, I just got to know the bug. Thanks for updating me with Travis Cl .